### PR TITLE
Document how to view application-specific logs throug Android Studio/adb

### PR DESCRIPTION
**Pull request by badboy**
Can anyone with a working Android Studio installation (cc @mdboom, @Dexterp37) verify the steps for Android Studio work (and do they work for any app or only for apps that are currently open as a project in Android Studio?)?

### Pull Request checklist ###

- [ ] **Quality**: This PR builds and tests run cleanly
  - `make test` runs without emitting any warnings
  - `make lint` runs without emitting any errors
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Changelog**: This PR includes a changelog entry or an explanation of why it does not need one
  - Any breaking changes to language binding APIs are noted explicitly

### CODE REVIEW

File: docs/user/debugging.md [hunk 1]
```diff
@@ -34,3 +34,49 @@ adb shell am start -n org.mozilla.samples.glean/mozilla.components.service.glean
 - There are a couple different ways in which to send pings through the GleanDebugActivity.
     1. You can use the `GleanDebugActivity` in order to tag pings and trigger them manually using the UI.  This should always produce a ping with all required fields.
     2. You can use the `GleanDebugActivity` to tag _and_ send pings.  This has the side effect of potentially sending a ping which does not include all fields because `sendPings` triggers pings to be sent before certain application behaviors can occur which would record that information.  For example, `duration` is not calculated or included in a baseline ping sent with `sendPing` because it forces the ping to be sent before the `duration` metric has been recorded.
+
+## Glean Log messages
+
+Glean logs warnings and errors through the Android logging framework.
+When running a Glean-powered app in the Android emulator or on a device connected to your computer via cable, there are several ways to read the log output
```
Comment (travis79) [inline docs/user/debugging.md]:
```suggestion
When running a Glean-powered app in the Android emulator or on a device connected to your computer via cable, there are several ways to read the log output.
```

File: docs/user/debugging.md [hunk 2]
```diff
@@ -34,3 +34,49 @@ adb shell am start -n org.mozilla.samples.glean/mozilla.components.service.glean
...
+```
+
+This is the unfiltered output of all log messages.
+You can match for `glean` using grep:
+
+```
+adb logcat | grep -i glean
+```
+
+A simpler way to filter for only the application to debug is using [pidcat][], a wrapper around `adb`, which adds colors and proper filtering by application ID and log level.
```
Comment (travis79) [inline docs/user/debugging.md]:
```suggestion
A simple way to filter for only the application that is being debugged is by using [pidcat][], a wrapper around `adb`, which adds colors and proper filtering by application ID and log level.
```

File: docs/user/debugging.md [hunk 3]
```diff
@@ -34,3 +34,49 @@ adb shell am start -n org.mozilla.samples.glean/mozilla.components.service.glean
...
+
+1. Run an app on your device.
+2. Click **View > Tool Windows > Logcat** (or click **Logcat** in the tool window bar).
+
+The Logcat window will show all log messages and allows to filter those by the application ID.
+Select the application ID of the product you're debugging.
+
+More information can be found in the [View Logs with Logcat][] help article.
+
+[View Logs with Logcat]: https://developer.android.com/studio/debug/am-logcat
```
Comment (mdboom) [inline docs/user/debugging.md]: It is worth mentioning that you can also filter on "Glean"?

Comment (badboy) [reply docs/user/debugging.md]: Good point. Let me add that.

**Pull request merged**